net/dns/nm.go

```diff
@@ -302,7 +302,7 @@ func (m *nmManager) GetBaseConfig() (OSConfig, error) {
 	for _, cfg := range cfgs {
 		if name, ok := cfg["interface"]; ok {
 			if s, ok := name.Value().(string); ok && s == m.interfaceName {
-				// Config for the taislcale interface, skip.
+				// Config for the tailscale interface, skip.
```

Comment:
Sometimes the brand is intentionally misspelled--this didn't seem to be one of those cases.

Reply:
I don't think we ever intentionally misspell it. When we talk about the daemon process we write `tailscaled` all lowercase. Or sometimes we talk about the `tailscale0` network interface. But this looks like it was certainly a mistake.